src/parser/jobs/nin/Ninjutsu.js

```diff
@@ -64,5 +119,57 @@ export default class Ninjutsu extends Module {
 				</Fragment>,
 			}))
 		}
+
+		const {badTcjs, badAoes, badStds} = this._appraiseDotonCasts()
+		if (badTcjs > 0) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.DOTON.icon,
+				content: <Fragment>
+					<Trans id="nin.ninjutsu.suggestions.tcj-doton.content">Avoid using <ActionLink {...ACTIONS.DOTON}/> under <ActionLink {...ACTIONS.TEN_CHI_JIN}/> unless at least {TCJ_DOTON_TICK_TARGET} ticks will hit or you&apos;re up against multiple targets. On a single target that&apos;s about to jump or move, using the <ActionLink {...ACTIONS.SUITON}/> combo will do more damage even if <ActionLink {...ACTIONS.TRICK_ATTACK}/> is on cooldown.</Trans>
```

Comment:
You can get away with apostrophes w/o the xml escape now.

Reply:
Oh neat. I'll stop using it then.